### DISCUSSION TITLE
Why is this stupid subscription so hard to cancel.

### DISCUSSION THREAD

**Discussion by Alexsiegel94** [community/community#106676] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

This app/website has no clear way to cancel my account. How do I cancel the account without going through endless “how to do this” threads and informational bull crap. I just want a link that goes straight to cancel

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Alexsiegel94! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, can you be a bit more clear about what you are trying to cancel? https://docs.github.com/en/search?query=cancel+subscription.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by Olayiwolaaa** [comment]
# check account settings
